app/backend/src/couchers/models/__init__.py

```diff
@@ -0,0 +1 @@
+from .User import User
```

Comment:
Please import from `couchers.models.user`, not relative

Reply:
I'm not a fan of code in `__init__.py`. Can we leave the file empty please?

Reply:
OK. I don't see a problem with keeping `__init__.py` clean. It makes it easier to see where the models really are if we don't do import stuff in init.

Reply:
We could empty it (and change imports to it) in a later PR.

Reply:
sounds good to empty it in a later PR :)